browser-interface/scripts/publish.ts

```diff
@@ -7,8 +7,18 @@ const DIST_ROOT = resolve(__dirname, "../dist")
 
 async function main() {
   await checkFiles()
-  if (process.env.CIRCLE_BRANCH == "master") {
+  
+  const branchName = process.env.CIRCLE_BRANCH;
+
+  if (branchName === "master") {
     await publish(["latest"], "public", DIST_ROOT)
+  } else {
+    const circleBuildNum = process.env.CIRCLE_BUILD_NUM;  
+    let match = /[^a-zA-Z0-9-]/g;
+    const cleanedBranchName = branchName!.replace(match, "-")
+    const tagName = `1.0.0.${circleBuildNum}${cleanedBranchName}`
+
+    await publish([tagName], "public", DIST_ROOT);
```

Comment:
What is the need behind this change? it will overload the package registry. To test the explorer this _wont_ be needed as the artifacts are uploaded to S3 (which is cheaper and easier to manage than NPM)

Reply:
As we recently changed the `update-renderer` make command kernel side, I had a bobazo and forgot we changed the overall approach. Closing in 3...2...1...